internal/rpc/starknet/errors.go

```diff
@@ -9,6 +9,30 @@ func (s *StarkNetError) Error() string {
 	return s.Message
 }
 
+func NewUnexpectedError() *StarkNetError {
```

Comment:
Should the errors be variables instead of functions? I'm not familiar with Go's idiom in this case.

Reply:
Yep, definitely ✅ https://github.com/NethermindEth/juno/pull/360/commits/e2c4e4d26b68cdd451351f8fd5137477de3d32ae